classes/system/ceilometer/server/cluster.yml

```diff
@@ -0,0 +1,14 @@
+classes:
+- service.ceilometer.server.cluster
+- service.haproxy.proxy.single
+- service.keepalived.cluster.single
+parameters:
+  _param:
+    ceilometer_influxdb_user: ceilometer
+    ceilometer_influxdb_password: ${_param:stacklight_influxdb_password}
+    ceilometer_influxdb_database: ceilometer
+    ceilometer_influxdb_host: ${_param:monitoring_service_address}
+    ceilometer_influxdb_port: 8086
+    ceilometer_elasticsearch_enable: true
+    ceilometer_elasticsearch_port: 9200
+    ceilometer_elasticsearch_host: ${_param:monitoring_service_address}
```

Comment:
Having all these params here in classes/system/ceilometer suggest that all the clusters that include Ceilometer will use Ceilometer with the InfluxDB and Elasticsearch plugins. That sounds reasonable to me, but I just wanted to make sure that's what you had in mind as well.